From a925b42add943dcc210a1d44e4841a88264e4454 Mon Sep 17 00:00:00 2001
From: Patil, Rachna <rachna@ti.com>
Date: Thu, 29 Sep 2011 14:47:45 +0530
Subject: [PATCH 15/20] AM335X: Enable touchscreen driver in default am335x_evm_defconfig

This patch enables ti_tscadc touchscreen driver in the default
am335x_evm_deconfig.

Signed-off-by: Patil, Rachna <rachna@ti.com>
---
 arch/arm/configs/am335x_evm_defconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/am335x_evm_defconfig b/arch/arm/configs/am335x_evm_defconfig
index 1a7bfda..d69a03f 100644
--- a/arch/arm/configs/am335x_evm_defconfig
+++ b/arch/arm/configs/am335x_evm_defconfig
@@ -116,6 +116,7 @@ CONFIG_KEYBOARD_GPIO=y
 CONFIG_KEYBOARD_TWL4030=y
 CONFIG_INPUT_TOUCHSCREEN=y
 CONFIG_TOUCHSCREEN_ADS7846=y
+CONFIG_TOUCHSCREEN_TI_TSCADC=y
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_TWL4030_PWRBUTTON=y
 CONFIG_VT_HW_CONSOLE_BINDING=y
-- 
1.6.6.1

